File: tutorials/java/misc_revision/collections/HashMaps.java
import java.util.HashMap;

public class HashMaps {
    public static void main(String[] args) {
        HashMap<Integer, String> map = new HashMap<>();

        map.put(1, "Batman");
        map.put(2, "Joker");
        map.put(3, "Alfred");
        map.put(4, "Scarecrow");

        System.out.println("val at key 3 = " + map.get(3));

        for (Integer key : map.keySet()) {
            System.out.println(key + " : " + map.get(key));
        }
    }
}

File: tutorials/cpp/stl/containers/hashmaps.cpp
#include <iostream>
#include <map>

int main(void)
{   
    std::map<int, std::string> hashmap;

    hashmap[1] = "Batman";
    hashmap[2] = "Joker";
    hashmap[3] = "Riddler";
    hashmap[4] = "Scarecrow";

    std::cout << "map elements: \n";
    for (auto &entry : hashmap)
    {
        std::cout << entry.first << " : " << entry.second << std::endl;
    }

    return EXIT_SUCCESS;
}